charger/versicharge.go

```diff
@@ -102,11 +104,11 @@ func (wb *Versicharge) Status() (api.ChargeStatus, error) {
 	s := binary.BigEndian.Uint16(b)
 
 	switch s {
-	case 1: // Available
+	case 65: // Status A
 		return api.StatusA, nil
-	case 2, 5: // Preparing, Suspended EV, Suspended EVSE
+	case 66, 16945: // Status B, B1
 		return api.StatusB, nil
-	case 3, 4: // Charging
+	case 67, 17201: // Status C, C1
```

Comment:
Das sind doch eigentlich zwei Byte lange Strings, oder?
Die kann man inzwischen automatisch mappen lassen.

Reply:
Siehe https://github.com/evcc-io/evcc/pull/9142

Reply:
habe seit FW 2.128 (VersiCharge Gen3) ebenso das Problem mit der Fehlermeldung (charger: modbus: exception '2' (illegal data address), function '3'). Wann ist ein neues Release mit den o.a. Template-Versionen zu rechnen?
Habe ich in der Zwischenzeit die Möglichkeit die Änderungen selber vorzunehmen?

Reply:
C1 ist bei mir noch nie vorgekommen. Ich habe C1 Charging zugeordnet, falsch?

Reply:
Der scheint tatsächlich noch nicht zu laden: https://github.com/evcc-io/evcc/pull/9142/files#diff-bf98d5fab5bcfabded9769069e7318683f549ec91cff84c5f87a9ae3a3ca3d8dR53